k8s/namespaces/family-public-law/fpl-case-service/aat.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fpl-case-service
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/fpl/case-service:prod-5a4b25f5
+      ingressHost: fpl-case-service-aat.service.core-compute-aat.internal
+      environment:
+        MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE: health, info, threaddump
+        GATEWAY_URL: https://gateway-ccd.aat.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.aat.platform.hmcts.net
+        SPRING_SECURITY_ENABLED: false
+        TESTING_SUPPORT_ENABLED: true
+    global:
+      environment: aat
+      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      enableKeyVaults: true
```

Comment:
These are not needed, they come from platform defaults when using kustomize.

```suggestion
```

Reply:
sorted ... added the reference in environment specific customisation.yaml
patchesStrategicMerge:
- ../../../namespaces/family-public-law/fpl-case-service/aat.yaml